src/modules/0.0.9/aws_elb/index.ts

```diff
@@ -41,7 +41,7 @@ export const AwsElbModule: Module2 = new Module2({
           out.actionType = (a.Type as ActionTypeEnum);
           out.targetGroup =  await AwsElbModule.mappers.targetGroup.db.read(ctx, a?.TargetGroupArn) ??
             await AwsElbModule.mappers.targetGroup.cloud.read(ctx, a?.TargetGroupArn);
-          if (!out.targetGroup) throw new Error('Target groups need to be loaded first');
+          if (!out.targetGroup) return undefined;
```

Comment:
If the target group can be loaded out of order, wouldn't this change break in legitimate situations? That's why I didn't change this one.

Reply:
Hmmm I see your point. But, if it is in the wrong order it means it will not find it on `db` but if that's true, it will be `undefined` and we will default to the `cloud` path where it should exists if any, right?

Reply:
That makes sense, but then I wonder what this was guarding against in the first place?

Reply:
Well the possibility that for some reason the target group is not  in `cloud` either. That mean something is wrong and misconfigured. Because a listener cannot be created without a target group to forward to.

Reply:
Okay, so we enforce the target group via non-nullable foreign key relations in the database, right?

So any load balancer in the cloud without a target group is some misconfiguration on the other side, then? There is no situation where a race condition from creating both of them in the not-too-distant past would cause one API to return null while the other succeeds?

In that case I don't have an issue with this, but if there is some race condition situation possible, can we add a singular delayed retry, say 10-20 seconds later, to try to catch that?